flask_resources/context.py

```diff
@@ -72,3 +73,19 @@ def inner(*args, **kwargs):
             return f(*args, **kwargs)
 
     return inner
+
+
+def with_route(f):
+    """Wrapper to extract route arguments into resource context.
+
+    NOTE: This removes the need for *args, **kwargs passing in Resource methods
+    TODO: This should be temporary because the whole decorator approach should be
```

Comment:
Agree. Could you open an issue?

Reply:
Good. Wanted that double check. I created the issue:  https://github.com/inveniosoftware/flask-resources/issues/49

Reply:
IMHO the issue does not represent the comment 😅 There is no reference to "decorators". Mainly I do not fully agree with removing the `*args` and `**kwargs` so it feels like is the only solution there :(

Plus the `decorator` approach, needs to be reviewed in general (for all generic decorators).

Reply:
Ha my bad I read that wrong :sweat_smile: ! I made a point about it in https://github.com/inveniosoftware/flask-resources/issues/2